docs/create-packages/Symbol-Packages-V2.md

```diff
@@ -0,0 +1,101 @@
+---
+title: How to publish NuGet symbol packages using the new symbol package format '.snupkg'| Microsoft Docs
+author:
+- cristinamanu
+- kraigb
+ms.author:
+- cristinamanu
+- kraigb
+manager: skofman
+ms.date: 10/30/2018
+ms.topic: reference
+ms.prod: nuget
+ms.technology: null
+description: How to create NuGet symbol packages (snupkg).
+keywords: NuGet symbol packages, NuGet package debugging, supporting NuGet debugging, package symbols, symbol package conventions
+ms.reviewer:
+- anangaur
+- karann
+
+
+# Creating symbol packages (.snupkg)
+> [!Important]
+> This feature is available only with [nuget.exe v4.9.0 or above](https://www.nuget.org/downloads) or [dotnet.exe v2.2.0 or above](https://www.microsoft.com/net/download/dotnet-core/2.2), which implement the required [NuGet protocols](../api/nuget-protocols.md).
+> The new format for the symbol packages in .snupkg. The previous format .symbols.nupkg is still supported but only for compatibility reasons (see [Symbol-Packages](Symbol-Packages.md)).
```

Comment:
.. The new format for the symbol packages **is** .snupkg

Reply:
Updated